content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -59,41 +81,152 @@ const result = await prisma.posts.findMany({
+| Query          | Match? | Description                                            |
+| :------------- | :----- | :----------------------------------------------------- |
+| `+fox +dog`    | Yes    | The text contains 'fox' and 'dog'                      |
+| `+dog +fox`    | Yes    | The text contains 'dog' and 'fox'                      |
+| `+dog -cat`    | No     | The text contains 'dog' but not 'cat'                  |
+| `-cat`         | Yes    | 'cat' is not in the text                               |
+| `fox dog`      | Yes    | The text contains 'fox' or 'cat'                       |
+| `-cat -pig`    | No     | The text does not contain 'cat' or 'pig'               |
+| `quic*`        | Yes    | The text contains a word starting with 'quic'          |
+| `quick fox @2` | Yes    | 'fox' starts within a 2 word distance of 'quick'       |
+| `fox dog @2`   | No     | 'dog' does not start within a 2 word distance of 'fox' |
+| `"jumps over"` | Yes    | The text contains the whole phrase 'jumps over'        |
 
-## Adding Indexes
+MySQL also has `>`, `<` and `~` operators for altering the ranking order of search results. As an example, consider the following two records:
 
-To speed up your full-text queries, you should add an index to your database.
-Prisma Migrate currently doesn't support adding search indices, but you can
-easily add one yourself.
+**1. "The quick brown fox jumps over the lazy dog"**
+
+**2. "The quick brown fox jumps over the lazy cat"**
+
+| Query             | Result                   | Description                                                                                             |
+| :---------------- | :----------------------- | :------------------------------------------------------------------------------------------------------ |
+| `fox ~cat`        | Return 1. first, then 2. | Return all records containing 'fox', but rank records containing 'cat' lower                            |
+| `fox (<cat >dog)` | Return 1. first, then 2. | Return all records containing 'fox', but rank records containing 'cat' lower than rows containing 'dog' |
+
+For the full range of supported operations, see the [MySQL full text search documentation](https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html).
+
+## Adding indexes
+
+### PostgreSQL
+
+To speed up your full-text queries, you should add an index to your database. Prisma Migrate currently doesn't support adding search indices in PostgreSQL, so this should be added in SQL. For example, the following SQL statement would add an index called `post_body_index` on the `posts_body_index` column of the `posts` table:
 
 ```sql
 CREATE INDEX post_body_index ON posts USING GIN (body);
 ```
 
-You can continue using Prisma Migrate as you were before, it will ignore indexes
-that it doesn't know about.
+For further information, see the [Postgres documentation on indexes](https://www.postgresql.org/docs/12/sql-createindex.html).
```

Comment:
```suggestion
For further information, see the [Postgres documentation on text search indexes](https://www.postgresql.org/docs/14/textsearch-tables.html#TEXTSEARCH-TABLES-INDEX).
```

Reply:
Thanks, agree that that section would be better. Can I check whether we want 12 or 14 though?

Reply:
Do we link usually link to the minimum version we support or the latest versions of the databases? 

Switching to 12 would still work with the link

Reply:
Thanks. Updated to the v12 version of this link for now, happy to change if necessary.

Reply:
This would be worth bringing up with Product/ Docs team on the version to include in the database when we link out content.